rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/interactor/CovidPatientProfileFragmentInteractor.java

```diff
@@ -1,10 +1,24 @@
 package io.ona.rdt.interactor;
 
 import io.ona.rdt.util.CovidFormLauncherAndSaver;
+import io.ona.rdt.util.CovidFormSaver;
+import io.ona.rdt.util.FormSaver;
 
 /**
  * Created by Vincent Karuri on 15/06/2020
  */
 public class CovidPatientProfileFragmentInteractor extends CovidFormLauncherAndSaver {
 
+    @Override
+    protected FormSaver getFormSaver() {
+        if (formSaver == null) {
+            formSaver = createFormSaver();
+        }
+        return formSaver;
+    }
+
+    @Override
+    protected FormSaver createFormSaver() {
+        return new CovidFormSaver();
+    }
```

Comment:
Is this needed?

Reply:
Sorry, it is not needed. I'll remove it. It was by mistake.